app/client/src/modules/ui-builder/ui/wds/WDSSelectWidget/config/propertyPaneConfig/validations/defaultOptionValueValidation.ts

```
@@ -0,0 +1,62 @@
+  let isValid;
+  let parsed;
+  let message = { name: "", message: "" };
+  /*
+   * Function to check if the object has `label` and `value`
+   */
+  // TODO: Fix this the next time the file is edited
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const hasLabelValue = (obj: any) => {
+    return (
+      _.isPlainObject(value) &&
+      obj.hasOwnProperty("label") &&
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)